src/coll/options.rs

```diff
@@ -806,7 +809,8 @@ pub struct FindOneOptions {
     pub show_record_id: Option<bool>,
 
     /// The number of documents to skip before counting.
-    pub skip: Option<i64>,
+    #[serde(serialize_with = "crate::bson_util::serialize_u64_option_as_i64")]
```

Comment:
little nit/question: is there a reason the helper function is fully qualified here with the `crate::bson_util` prefix but that's omitted in usages above? (is it doing anything different?)

Reply:
No reason, probably just because I didn't have autocomplete here. Removed.